src/material-experimental/mdc-tabs/ink-bar.ts

```diff
@@ -89,10 +88,23 @@ export class MatInkBarFoundation {
     }
   };
 
-  constructor(elementRef: ElementRef<HTMLElement>, document: Document) {
-    this._element = elementRef.nativeElement;
+  /**
+   * Whether the indicator should be appended to the content, which will cause the ink bar
+   * to match the width of the content rather than the tab host element.
+   */
+  get fitToContent(): boolean { return this._fitToContent; }
+  set fitToContent(fitToContent: boolean) {
```

Comment:
I don't think we need the getter here and the setter could be turned into a function. Since this is a private class we don't need to deal with getters and setters.

Reply:
I just tried that approach but I'm not sure I feel as comfortable with it. There are two conditions I need to meet:

- Let clients see what the current value is
- Let clients set a new value, and the foundation reacts to it

To satisfy the first condition, there needs to be a public value `fitToContent`. However, it would not be clear to clients that in order to change this value, they should not write directly to it, but instead use a `setFitToContent` function.

If clients didn't need to read the value, I'd agree that a setter would work, but in this case I think the getter/setter pattern is ideal

Reply:
Kristiyan is saying that this component (ink bar) is not part of the public API, and as such doesn't need a getter/setter (which we only need for Angular inputs) compared to using functions

Reply:
Sorry, I see that my explanation may be vague regarding this. 

When `fitInkBarToContent` is provided as an input to the `MatTabGroup` and `MatTabNav`, they store the value into the `MatInkBarFoundation`. This will be the source of truth for it's value (not cached in the tab group or nav). When the value is set, the foundation needs to render again, hence the need for a setter (this can be either a native setter or `setFitToContent`). 

When our users want to access the property `fitInkBarToContent`, we give them the value stored in `MatInkBarFoundation`. There are two ways we can do this, expose the property `fitToContent`, or provide a `getFitToContent` function. If we expose the property, it's not clear from the `MatTabGroup`/`MatTabNav` that it should only use that property for getting, not for setting. In the future, we may make a change and not realize the setter _must_ be used, not the property.

If this doesn't make sense, I may be missing what you are proposing. Let's chat in the next team meeting

Reply:
Andrew and I chatted, he just didn't follow that we meant do e.g. `getFitToContent` and `setFitToContent` instead